<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicVideo</class>
 <widget class="QWidget" name="musicVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="fileButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="webLink"/>
    </item>
    <item>
     <widget class="QPushButton" name="enterButton">
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>or enter URL link</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
